web/public/templates/errors/404.dust

```diff
@@ -2,8 +2,21 @@
 {<body}
   {>"common/header" /}
   <div class="container">
-    <div>
-      {@i18n key="Page not found"/}
+    <div class="col-xs-12 col-sm-10 col-sm-offset-1 text-center">
+      <div class="ninja-container">
+        <div class="ninja"></div>
+      </div>
+      <div class="page-not-found-caption">
+        <h4>{@i18n key="Oops! This is not the page you are looking for."/}</h4>
```

Comment:
Anything which uses the i18n feature will have to be added to [the messages.po file here](https://github.com/CoderDojo/cp-zen-platform/blob/master/web/locale/en_US/messages.po).

The msgid is the same as the msgstr for the English file (and you only need to do English because the rest is automatically updated through our localisation platform).